LocalPackages/MallocInterposerC/Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version: 6.1
```

Comment:
Should move to "swift-tools-version: 5.10" everywhere, as we still support 5.10.